decrypted.txt

```diff
@@ -0,0 +1,3 @@
+hello world!
```

Comment:
one thing i haven't figured out is how to keep capitalized characters through encrypt and decrypt process